src/ert/_c_wrappers/enkf/config/surface_config.py

```diff
@@ -31,7 +32,13 @@ class SurfaceConfig(ParameterConfig):
     output_file: Path
     base_surface_path: str
 
-    def load(self, run_path: Path, real_nr: int, ensemble: EnsembleAccessor):
```

Comment:
Is this a good idea? Maybe we should split this into two functions, a `load_from_file` function and a `sample` function, so it is more explicit. Means some extra work, but think it might have benefits in the long run, what do you think? Goes for all the configs. Might require some enum instead of `forward_init`, like `initiation_type` ( SAMPLE, LOAD_AT_START or LOAD_AFTER_FORWARD_MODEL (or something like that)). What do you think?

Reply:
Ended up starting to implement what you suggested by splitting it up into `load` and `sample`